data-structures-linked-lists/insert-next.js

```diff
@@ -0,0 +1,11 @@
+/* global LinkedList */ // eslint-disable-line no-unused-vars
+/* exported insertNext */
+
+function insertNext(list, value) {
+  const newlist = list.next;
+  list.next = new LinkedList(value);
+  if (newlist !== null) {
+    list.next.next = newlist;
+  }
+  return list;
```

Comment:
`insertNext` should not return anything:

---

![image](https://user-images.githubusercontent.com/16143771/156838108-f67021bf-78af-4bde-b7d0-866ab559211f.png)